### PR TITLE
Create AI page

### PR DESCRIPTION
**Pull request by albertkol**
## Done

- Create AI page following this design: https://github.com/canonical-web-and-design/web-squad/issues/3575 and this docs: https://docs.google.com/document/d/1HP9YdHF11yVQ9SLl0VXhLezDZREsr1GHIKNi9wT1Gv0/edit?ts=5fb6b19b#

## QA

- Go to https://ubuntu-com-8970.demos.haus/ai and check against design or docs

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/3576

### CODE REVIEW

File: templates/ai/index.html [hunk 1]
```diff
@@ -0,0 +1,688 @@
...
+          }}
+      </li>
+    </ul>
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1>Develop</h1>
```
Comment (bethcollins92) [inline templates/ai/index.html:103]: I think these headings should be h2 going from the design, (there should only be one h1 on the page)
```suggestion
      <h2>Develop</h2>
```

File: templates/ai/index.html [hunk 2]
```diff
@@ -0,0 +1,688 @@
...
+      <h4>Lifecycle management of your data stack on Kubernetes</h4>
+      <ul>
+        <li>Make use of open source lifecycle management code and compound the benefit of
+          <a href="http://charmhub.io">200+ charm operators</a>.</li>
+        <li>Composable operators for data scientists to stand up and integrate the applications they need, on a laptop, workstation or cluster.</li>
+        <li>Install, configure, upgrade, remove.</li>
+      </ul>
+      <p>
+        <a href="http://charmed-kubeflow.io" class="p-button--positive p-link--external">Kubeflow operators</a>
+        <a href="/ai/contact-us?product=ai-index">Contact us&nbsp;&rsaquo;</a>
```
Comment (bethcollins92) [inline templates/ai/index.html:499]: Check this comment https://docs.google.com/document/d/1HP9YdHF11yVQ9SLl0VXhLezDZREsr1GHIKNi9wT1Gv0/edit?disco=AAAALCK26VU, it's a different contact us

File: templates/ai/index.html [hunk 3]
```diff
@@ -0,0 +1,688 @@
...
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1>Every Kubeflow operator you need</h1>
```
Comment (bethcollins92) [inline templates/ai/index.html:489]: All of these subheadings should be h2

File: templates/ai/index.html [hunk 4]
```diff
@@ -0,0 +1,688 @@
...
+              hi_def=True,
+              loading="auto|lazy",
+              attrs={"class":"p-heading-icon__img"}
+              ) | safe
+            }}
+            <h4 class="p-heading-icon__title">DOWNLOAD</h4>
+          </div>
+          <ul>
+            <li><a href="https://ubuntu.com/engage/starting-with-ai">Whitepaper: Get started with AI&nbsp;&rsaquo;</a></li>
+            <li><a href="https://drive.google.com/file/d/1it6EznImfYoV1fflc-ze8-3MhOce6wm8/view">Kubeflow Reference Architecture&nbsp;&rsaquo;</a></li>
```
Comment (bethcollins92) [inline templates/ai/index.html:639]: Double check this with marketing/Sam, I think this should be an asset link not a google drive link

**Review by bethcollins92 — CHANGES_REQUESTED** [review]
Looking good! Just a few comments in the code 👍🏼

**Comment by spencerbygraves** [comment]
@albertkol thanks, a few changes from me please.

- All section titles should be h2 please, from Develop and Deploy onwards
- The icons in the Learn more section are the wrong size/stretched. As an example, you can see the correct size on [this page](https://ubuntu.com/kubernetes)
- Can we update the logo sizes please, I've listed the heights below

**Develop**
Jupyter = 64px
Keras = 32
scikit = 48
Tensorflow = 32
Pytorch = 30

**Deploy**
Seldon = 35
KFServing = 28
Onyx = 30
TensorRT = 48
XGBoost = 20

**ML operations**
Kubeflow = 32
mlflow = 40
Pachyderm = 38
Juju = 32

**Data lake**
Spark = 32
Kafka = 36
Cassandra = 64
Elastic = 40
Hadoop = 40

**Hardware control**
Nvidia = 32
AMD = 30
Intel = 28
arm = 24

**Portable to scale**
Server icon = 48
OpenStack = 48
aws = 40
Google = 28
Microsoft = 32

Thanks.

File: templates/ai/index.html (hunk 1 above)
Comment (albertkol) [reply templates/ai/index.html]: Fixed!

File: templates/ai/index.html (hunk 3 above)
Comment (albertkol) [reply templates/ai/index.html]: Fixed!

**Comment by albertkol** [comment]
@spencerbygraves design is ready for review again.

**Comment by spencerbygraves** [comment]
@albertkol thanks for updating the logos. The webinar and docs icons are still stretched vertically:

image

You can see them [here](https://assets.ubuntu.com/v1/3769b4a7-webinar-icon.svg) and [here](https://assets.ubuntu.com/v1/df54bb6a-document-open-icon.svg).

Thanks.

File: templates/ai/index.html (hunk 4 above)
Comment (albertkol) [reply templates/ai/index.html]: Changed to an assets link!

**Comment by bethcollins92** [comment]
Thanks @albertkol all LGTM 👍🏼

**Comment by albertkol** [comment]
@spencerbygraves design is ready for review again again.

**Comment by spencerbygraves** [comment]
Thanks for updating @albertkol 👍

**Review by bethcollins92 — APPROVED** [review]

**Pull request merged**